src/components/Onboarding/OnboardingTemplate.js

```
@@ -0,0 +1,55 @@
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import SkipButton from './SkipButton';
+import OnboardingImage from './OnboardingImage';
+import OnboardingDescription from './OnboardingDescription';
+
+const OnboardingTemplate = ({ navigation, imageSource, description, subText, skipButton }) => (
+  <View style={styles.container}>
+    {skipButton
+      ? skipButton
+      : <SkipButton onPress={() => navigation && navigation.replace ? navigation.replace('MainTabs') : null} />
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)